docs/Deploying-Pega-on-AKS.md

```diff
@@ -547,7 +645,7 @@ To log in to Pega Platform with this host name, assign the host name with the sa
 
 You can view the networking endpoint that is associated with your AKS deployment using the `kubectl` command.
 
-`$ kubectl get services --namespace mypega`
+`$ kubectl get services --namespace mypega-aks-demo`
```

Comment:
Maybe we should show example output?  Not required, just an idea.

Reply:
good point.  agree